mod_form.php

```diff
@@ -111,17 +113,28 @@ public function definition() {
         }
 
         // Load final quiz choices.
-        $quizchoices = array(0 => 'None');
-        if ($quizes = $DB->get_records_select('quiz', 'course = :course_id', array('course_id' => $id) )) {
+        $quizchoices = array('' => 'Select a Quiz');
+        if ($quizes = $DB->get_records_select('quiz', 'course = :course_id', array('course_id' => $id), '', 'id, name')) {
             foreach ($quizes as $quiz) {
                 $quizchoices[$quiz->id] = $quiz->name;
             }
         }
 
+        // Load course assigments.
+        $assigmentschoices = array('' => 'Select an Activity Grade');
+        $assigments = $DB->get_records_select('grade_items', 'courseid = :course_id AND itemtype = :item_type',
```

Comment:
Minor - This variable will not include only `assignments` but also `quizzes`, `forums`, etc., any gradable modules.